### DISCUSSION TITLE
add github action

### DISCUSSION THREAD

**Discussion by tk-t0n0y** [projectdiscovery/pdtm#59] [Q&A]
add github action script that will auto add all binary to [pdtm](https://github.com/projectdiscovery/pdtm) main . and use cron for update binary files regualrly  for fast clone .  it is hard to wait for fully install all tools so you can add binary for fast clone for disposable linux user

**Comment by ehsandeep** [comment]
@tk-t0n0y pdtm already and only uses binary files to install projects; updates are automatic using GitHub webhook upon project release so cron update is not needed.
